src/SerEvaluator.cs

```diff
@@ -194,10 +194,8 @@ public override Task<Capabilities> GetCapabilities(Empty request, ServerCallCont
                         var qrshub = new QlikQrsHub(onDemandConfig.Connection.ServerUri, tmpsession.Cookie);
                         var qrsResult = qrshub.SendRequestAsync($"app/{userParameter.AppId}", HttpMethod.Get).Result;
                         logger.Trace($"appResult:{qrsResult}");
-                        var hubInfo = JsonConvert.DeserializeObject<HubInfo>(qrsResult);
-                        if (hubInfo == null)
-                            throw new Exception($"No app owner for app id {userParameter.AppId} found.");
-                        userParameter.DomainUser = new DomainUser($"{hubInfo.Owner.UserDirectory}\\{hubInfo.Owner.UserId}");
+                        dynamic jObject = JObject.Parse(qrsResult);
+                        userParameter.DomainUser = new DomainUser($"{jObject?.owner?.userDirectory}\\{jObject?.owner?.userId}");
```

Comment:
please add at least a warn message if the jObject is null or .owner or userid,... is NULL